components/xwiki-platform-users/xwiki-platform-users-api/pom.xml

```diff
@@ -63,6 +63,11 @@
       <version>${xwiki.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.commons</groupId>
+      <artifactId>xwiki-commons-component-api</artifactId>
+      <version>7.0.1</version>
```

Comment:
Don't hardcode versions like this, use `${xwiki.version}`.

This makes it easier to upgrade the XWiki version, since it only requires changing one parameter in one place, instead of searching for all the individual dependencies in all the pom files.